index.html

```diff
@@ -22,7 +22,7 @@
     <link rel="icon" type="image/png" sizes="16x16" href="/public/icons/favicon-16x16.png">
 
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio</title>
+    <title>Raia</title>
   </head>
   <body>
```

Comment:
Update the title to reflect the individual's name for a personalized touch and consistency across the portfolio.